DesignKit/Source/Colors.swift

```diff
@@ -68,4 +68,11 @@ public protocol Colors {
     /// - Avatars default states that include first name letter
     var namesAndAvatars: [ColorType] { get }
     
+    // MARK: - Others colors
+    
+    /// White
+    var white: ColorType { get }
```

Comment:
The property name should reflect the purpose. It seems that this color is not used.

Reply:
We should also create another protocol to handle other colors. `Colors.swift` handles only colors from Figma Compound. We need to create `public protocol OtherColors` in a separated file and add `otherColors` property on `ThemeV2` and `ThemeSwiftUIType`.

Reply:
I removed here, they was no longuer necessary. For the protocol, I am looking how colors was handle in former project, I'll work on it later.